{"uid":"8c7c909eeda5f79c","name":"Archive a card","fullName":"tests.regression_ui.test_ui_regression.TestUIRegression#test_archive_card","historyId":"4151bed777d70c35651f4626d00b86b5","time":{"start":1735582847474,"stop":1735582879995,"duration":32521},"status":"broken","statusMessage":"TypeError: not all arguments converted during string formatting","statusTrace":"self = <tests.regression_ui.test_ui_regression.TestUIRegression object at 0x7ff7e89a66f0>\ndriver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"ed174e934680b735191b26b3da8e5771\")>\n\n    @pytest.mark.TC000\n    @pytest.mark.TC001\n    @allure.title('Login Functionality')\n    def test_login(self, driver):\n        with allure.step('Navigate to the Trello login page'):\n            driver.get(self.SCC.URL_LOGIN)\n            time.sleep(3)\n        with allure.step('Enter valid credentials (email, password, and 6-digits verification code)'):\n            try:\n                el_user_name = WebDriverWait(driver, 60).until(EC.element_to_be_clickable(\n                    (By.XPATH, self.SCC.LoginPage.USER_NAME_FIELD)\n                ))\n                el_user_name.click()\n                el_user_name.send_keys(USER_EMAIL)\n    \n                driver.find_element(By.XPATH, self.SCC.LoginPage.CONTINUE_BUTTON).click()\n                el_password = WebDriverWait(driver, 60).until(EC.element_to_be_clickable(\n                    (By.XPATH, self.SCC.LoginPage.PASSWORD_FIELD)\n                ))\n                el_password.click()\n                el_password.send_keys(USER_PASSWORD)\n    \n                driver.find_element(By.XPATH, self.SCC.LoginPage.LOGIN_BUTTON).click()\n                time.sleep(3)\n                el_six_otp_code = WebDriverWait(driver, 60).until(EC.element_to_be_clickable(\n                    (By.XPATH, self.SCC.LoginPage.SIX_OTP_CODE_FIELD)\n                ))\n                el_six_otp_code.click()\n                el_six_otp_code.send_keys(self.otp_auth())\n                time.sleep(10)\n>               WebDriverWait(driver, 10).until(EC.url_matches(self.SCC.HOME_URL))\n\ntests/regression_ui/test_ui_regression.py:84: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"ed174e934680b735191b26b3da8e5771\")>\nmethod = <function url_matches.<locals>._predicate at 0x7ff7e889a2a0>\nmessage = ''\n\n    def until(self, method: Callable[[D], Union[Literal[False], T]], message: str = \"\") -> T:\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            if time.monotonic() > end_time:\n                break\n            time.sleep(self._poll)\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message:\n\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/site-packages/selenium/webdriver/support/wait.py:105: TimeoutException\n\nDuring handling of the above exception, another exception occurred:\n\nself = <tests.regression_ui.test_ui_regression.TestUIRegression object at 0x7ff7e89a66f0>\ndriver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"ed174e934680b735191b26b3da8e5771\")>\n\n    @pytest.mark.TC000\n    @pytest.mark.TC006\n    @allure.title('Archive a card')\n    def test_archive_card(self, driver):\n        time.sleep(5)\n        with allure.step('Log in to Trello account'):\n>           self.test_login(driver)\n\ntests/regression_ui/test_ui_regression.py:188: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \ntests/regression_ui/test_ui_regression.py:88: in test_login\n    log.info(\"An error occurred: \", str(e))\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py:2216: in info\n    root.info(msg, *args, **kwargs)\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py:1539: in info\n    self._log(INFO, msg, args, **kwargs)\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py:1684: in _log\n    self.handle(record)\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py:1700: in handle\n    self.callHandlers(record)\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py:1762: in callHandlers\n    hdlr.handle(record)\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py:1028: in handle\n    self.emit(record)\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/site-packages/_pytest/logging.py:386: in emit\n    super().emit(record)\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py:1168: in emit\n    self.handleError(record)\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py:1160: in emit\n    msg = self.format(record)\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py:999: in format\n    return fmt.format(record)\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/site-packages/_pytest/logging.py:139: in format\n    return super().format(record)\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py:703: in format\n    record.message = record.getMessage()\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <LogRecord: root, 20, /home/runner/work/pythonSelenium/pythonSelenium/tests/regression_ui/test_ui_regression.py, 88, \"An error occurred: \">\n\n    def getMessage(self):\n        \"\"\"\n        Return the message for this LogRecord.\n    \n        Return the message for this LogRecord after merging any user-supplied\n        arguments with the message.\n        \"\"\"\n        msg = str(self.msg)\n        if self.args:\n>           msg = msg % self.args\nE           TypeError: not all arguments converted during string formatting\n\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py:392: TypeError","flaky":false,"newFailed":false,"newBroken":true,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"driver","time":{"start":1735582847208,"stop":1735582847473,"duration":265},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"setup","time":{"start":1735582847474,"stop":1735582847474,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"testStage":{"status":"broken","statusMessage":"TypeError: not all arguments converted during string formatting","statusTrace":"self = <tests.regression_ui.test_ui_regression.TestUIRegression object at 0x7ff7e89a66f0>\ndriver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"ed174e934680b735191b26b3da8e5771\")>\n\n    @pytest.mark.TC000\n    @pytest.mark.TC001\n    @allure.title('Login Functionality')\n    def test_login(self, driver):\n        with allure.step('Navigate to the Trello login page'):\n            driver.get(self.SCC.URL_LOGIN)\n            time.sleep(3)\n        with allure.step('Enter valid credentials (email, password, and 6-digits verification code)'):\n            try:\n                el_user_name = WebDriverWait(driver, 60).until(EC.element_to_be_clickable(\n                    (By.XPATH, self.SCC.LoginPage.USER_NAME_FIELD)\n                ))\n                el_user_name.click()\n                el_user_name.send_keys(USER_EMAIL)\n    \n                driver.find_element(By.XPATH, self.SCC.LoginPage.CONTINUE_BUTTON).click()\n                el_password = WebDriverWait(driver, 60).until(EC.element_to_be_clickable(\n                    (By.XPATH, self.SCC.LoginPage.PASSWORD_FIELD)\n                ))\n                el_password.click()\n                el_password.send_keys(USER_PASSWORD)\n    \n                driver.find_element(By.XPATH, self.SCC.LoginPage.LOGIN_BUTTON).click()\n                time.sleep(3)\n                el_six_otp_code = WebDriverWait(driver, 60).until(EC.element_to_be_clickable(\n                    (By.XPATH, self.SCC.LoginPage.SIX_OTP_CODE_FIELD)\n                ))\n                el_six_otp_code.click()\n                el_six_otp_code.send_keys(self.otp_auth())\n                time.sleep(10)\n>               WebDriverWait(driver, 10).until(EC.url_matches(self.SCC.HOME_URL))\n\ntests/regression_ui/test_ui_regression.py:84: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"ed174e934680b735191b26b3da8e5771\")>\nmethod = <function url_matches.<locals>._predicate at 0x7ff7e889a2a0>\nmessage = ''\n\n    def until(self, method: Callable[[D], Union[Literal[False], T]], message: str = \"\") -> T:\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            if time.monotonic() > end_time:\n                break\n            time.sleep(self._poll)\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message:\n\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/site-packages/selenium/webdriver/support/wait.py:105: TimeoutException\n\nDuring handling of the above exception, another exception occurred:\n\nself = <tests.regression_ui.test_ui_regression.TestUIRegression object at 0x7ff7e89a66f0>\ndriver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"ed174e934680b735191b26b3da8e5771\")>\n\n    @pytest.mark.TC000\n    @pytest.mark.TC006\n    @allure.title('Archive a card')\n    def test_archive_card(self, driver):\n        time.sleep(5)\n        with allure.step('Log in to Trello account'):\n>           self.test_login(driver)\n\ntests/regression_ui/test_ui_regression.py:188: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \ntests/regression_ui/test_ui_regression.py:88: in test_login\n    log.info(\"An error occurred: \", str(e))\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py:2216: in info\n    root.info(msg, *args, **kwargs)\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py:1539: in info\n    self._log(INFO, msg, args, **kwargs)\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py:1684: in _log\n    self.handle(record)\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py:1700: in handle\n    self.callHandlers(record)\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py:1762: in callHandlers\n    hdlr.handle(record)\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py:1028: in handle\n    self.emit(record)\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/site-packages/_pytest/logging.py:386: in emit\n    super().emit(record)\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py:1168: in emit\n    self.handleError(record)\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py:1160: in emit\n    msg = self.format(record)\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py:999: in format\n    return fmt.format(record)\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/site-packages/_pytest/logging.py:139: in format\n    return super().format(record)\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py:703: in format\n    record.message = record.getMessage()\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <LogRecord: root, 20, /home/runner/work/pythonSelenium/pythonSelenium/tests/regression_ui/test_ui_regression.py, 88, \"An error occurred: \">\n\n    def getMessage(self):\n        \"\"\"\n        Return the message for this LogRecord.\n    \n        Return the message for this LogRecord after merging any user-supplied\n        arguments with the message.\n        \"\"\"\n        msg = str(self.msg)\n        if self.args:\n>           msg = msg % self.args\nE           TypeError: not all arguments converted during string formatting\n\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py:392: TypeError","steps":[{"name":"Log in to Trello account","time":{"start":1735582852474,"stop":1735582879994,"duration":27520},"status":"broken","statusMessage":"TypeError: not all arguments converted during string formatting\n","statusTrace":"  File \"/home/runner/work/pythonSelenium/pythonSelenium/tests/regression_ui/test_ui_regression.py\", line 188, in test_archive_card\n    self.test_login(driver)\n  File \"/home/runner/work/pythonSelenium/pythonSelenium/tests/regression_ui/test_ui_regression.py\", line 88, in test_login\n    log.info(\"An error occurred: \", str(e))\n  File \"/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py\", line 2216, in info\n    root.info(msg, *args, **kwargs)\n  File \"/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py\", line 1539, in info\n    self._log(INFO, msg, args, **kwargs)\n  File \"/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py\", line 1684, in _log\n    self.handle(record)\n  File \"/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py\", line 1700, in handle\n    self.callHandlers(record)\n  File \"/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py\", line 1762, in callHandlers\n    hdlr.handle(record)\n  File \"/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py\", line 1028, in handle\n    self.emit(record)\n  File \"/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/site-packages/_pytest/logging.py\", line 386, in emit\n    super().emit(record)\n  File \"/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py\", line 1168, in emit\n    self.handleError(record)\n  File \"/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py\", line 1160, in emit\n    msg = self.format(record)\n          ^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py\", line 999, in format\n    return fmt.format(record)\n           ^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/site-packages/_pytest/logging.py\", line 139, in format\n    return super().format(record)\n           ^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py\", line 703, in format\n    record.message = record.getMessage()\n                     ^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py\", line 392, in getMessage\n    msg = msg % self.args\n          ~~~~^~~~~~~~~~~\n","steps":[{"name":"Navigate to the Trello login page","time":{"start":1735582852474,"stop":1735582855785,"duration":3311},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"Enter valid credentials (email, password, and 6-digits verification code)","time":{"start":1735582855785,"stop":1735582879992,"duration":24207},"status":"broken","statusMessage":"TypeError: not all arguments converted during string formatting\n","statusTrace":"  File \"/home/runner/work/pythonSelenium/pythonSelenium/tests/regression_ui/test_ui_regression.py\", line 88, in test_login\n    log.info(\"An error occurred: \", str(e))\n  File \"/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py\", line 2216, in info\n    root.info(msg, *args, **kwargs)\n  File \"/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py\", line 1539, in info\n    self._log(INFO, msg, args, **kwargs)\n  File \"/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py\", line 1684, in _log\n    self.handle(record)\n  File \"/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py\", line 1700, in handle\n    self.callHandlers(record)\n  File \"/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py\", line 1762, in callHandlers\n    hdlr.handle(record)\n  File \"/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py\", line 1028, in handle\n    self.emit(record)\n  File \"/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/site-packages/_pytest/logging.py\", line 386, in emit\n    super().emit(record)\n  File \"/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py\", line 1168, in emit\n    self.handleError(record)\n  File \"/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py\", line 1160, in emit\n    msg = self.format(record)\n          ^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py\", line 999, in format\n    return fmt.format(record)\n           ^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/site-packages/_pytest/logging.py\", line 139, in format\n    return super().format(record)\n           ^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py\", line 703, in format\n    record.message = record.getMessage()\n                     ^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/logging/__init__.py\", line 392, in getMessage\n    msg = msg % self.args\n          ~~~~^~~~~~~~~~~\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false}],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":2,"attachmentsCount":0,"hasContent":true,"attachmentStep":false}],"attachments":[{"uid":"aa288aaf82065fee","name":"log","source":"aa288aaf82065fee.txt","type":"text/plain","size":63296}],"parameters":[],"shouldDisplayMessage":true,"stepsCount":3,"attachmentsCount":1,"hasContent":true,"attachmentStep":false},"afterStages":[{"name":"driver::0","time":{"start":1735582880152,"stop":1735582880224,"duration":72},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"labels":[{"name":"tag","value":"TC006"},{"name":"tag","value":"TC000"},{"name":"parentSuite","value":"tests.regression_ui"},{"name":"suite","value":"test_ui_regression"},{"name":"subSuite","value":"TestUIRegression"},{"name":"host","value":"fv-az1691-408"},{"name":"thread","value":"3624-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.regression_ui.test_ui_regression"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"history":{"statistic":{"failed":3,"broken":12,"skipped":0,"passed":39,"unknown":0,"total":54},"items":[{"uid":"b4bcdbd7c507b5ab","status":"passed","time":{"start":1735520859272,"stop":1735520891126,"duration":31854}},{"uid":"643c42e9d4d82917","status":"passed","time":{"start":1735496353392,"stop":1735496385411,"duration":32019}},{"uid":"6b7f1fb1c301cb6","status":"passed","time":{"start":1735434689560,"stop":1735434721444,"duration":31884}},{"uid":"6182c3469ba92620","status":"passed","time":{"start":1735410019015,"stop":1735410050963,"duration":31948}},{"uid":"c1557cda4a8e33b","status":"passed","time":{"start":1735347855664,"stop":1735347887429,"duration":31765}},{"uid":"954adf59b4913f18","status":"failed","statusDetails":"AssertionError: assert 1 == 0","time":{"start":1735323629386,"stop":1735323661146,"duration":31760}},{"uid":"3a99cab29fdc8164","status":"broken","statusDetails":"selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x55d44d68731a <unknown>\n#1 0x55d44d19d6e0 <unknown>\n#2 0x55d44d1ec3e6 <unknown>\n#3 0x55d44d1ec681 <unknown>\n#4 0x55d44d231b04 <unknown>\n#5 0x55d44d21048d <unknown>\n#6 0x55d44d22eed7 <unknown>\n#7 0x55d44d210203 <unknown>\n#8 0x55d44d1decc0 <unknown>\n#9 0x55d44d1dfc9e <unknown>\n#10 0x55d44d654d0b <unknown>\n#11 0x55d44d658c92 <unknown>\n#12 0x55d44d641b3c <unknown>\n#13 0x55d44d659807 <unknown>\n#14 0x55d44d6270df <unknown>\n#15 0x55d44d676578 <unknown>\n#16 0x55d44d676740 <unknown>\n#17 0x55d44d686196 <unknown>\n#18 0x7f80de89ca94 <unknown>\n#19 0x7f80de929c3c <unknown>","time":{"start":1735261620949,"stop":1735261653776,"duration":32827}},{"uid":"df0cdedbf8f7b71b","status":"passed","time":{"start":1735237228427,"stop":1735237260240,"duration":31813}},{"uid":"951a8163e12e15a","status":"passed","time":{"start":1735175103507,"stop":1735175135246,"duration":31739}},{"uid":"d226d7018243506d","status":"broken","statusDetails":"TypeError: not all arguments converted during string formatting","time":{"start":1735150876497,"stop":1735150908985,"duration":32488}},{"uid":"d17e727f3a76c1e4","status":"failed","statusDetails":"AssertionError: assert 1 == 0","time":{"start":1735088748609,"stop":1735088780384,"duration":31775}},{"uid":"7ea099cce60b2466","status":"passed","time":{"start":1735064419152,"stop":1735064450963,"duration":31811}},{"uid":"825b50257751efc3","status":"passed","time":{"start":1735002342502,"stop":1735002374320,"duration":31818}},{"uid":"20c1854dcd491f2a","status":"passed","time":{"start":1734978026823,"stop":1734978058674,"duration":31851}},{"uid":"b57764b64ee78d48","status":"passed","time":{"start":1734916024327,"stop":1734916056159,"duration":31832}},{"uid":"3c695bff913369c7","status":"passed","time":{"start":1734891544514,"stop":1734891576590,"duration":32076}},{"uid":"6941b1948d86c65c","status":"broken","statusDetails":"TypeError: not all arguments converted during string formatting","time":{"start":1734829996568,"stop":1734830029140,"duration":32572}},{"uid":"518caa36e8aab9d8","status":"passed","time":{"start":1734805149528,"stop":1734805181329,"duration":31801}},{"uid":"84fc8400ad3195c3","status":"broken","statusDetails":"TypeError: not all arguments converted during string formatting","time":{"start":1734743087487,"stop":1734743120055,"duration":32568}},{"uid":"94206452dcca79f2","status":"passed","time":{"start":1734718838227,"stop":1734718870082,"duration":31855}}]},"tags":["TC006","TC000"]},"source":"8c7c909eeda5f79c.json","parameterValues":[]}